source/guide_monitoror.rst

```diff
@@ -0,0 +1,100 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/monitoror.png
+      :align: center
+
+#########
+Monitoror
+#########
+
+.. tag_list::
+
+Monitoror_ is a wallboard monitoring app to monitor server status; monitor CI builds progress or even display critical values.
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+First we create an installation directory and then we need to configure the `web backend <webbackend_>`_.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mkdir monitoror
+  [isabell@stardust ~]$ uberspace web backend set / --http --port 8080
+  Set backend for / to port 8080; please make sure something is listening!
+  You can always check the status of your backend using "uberspace web backend list".
+  [isabell@stardust ~]$
+
+.. _webbackend: https://manual.uberspace.de/web-backends.html
+
+Installation
+============
+
+Like a lot of Go software, Monitoror is distributed as a single binary. Download Monitoror's latests `release with the following command.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ cd monitoror
+  [isabell@stardust monitoror]$ curl -sL -o monitoror $(curl -s https://api.github.com/repos/monitoror/monitoror/releases/latest | grep 'browser_download_url.*monitoror-linux-amd64' | cut -d: -f2,3 | tr -d \")
```

Comment:
Please refer to the latest version [like this](https://github.com/monitoror/monitoror/releases/download/3.1.0/monitoror-linux-amd64-3.1.0) here. This line is hard to understand and might break, if github changes anything.